src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -131,10 +131,67 @@ public PackageArchiveReader BuildPackage(PackageBuilder builder, string outputPa
                 PrintVerbose(outputPath, builder);
             }
 
+            if (_packArgs.InstallPackageToOutputPath)
```

Comment:
This is incorrect - you need to do either WriteResolvedNusPecToPackageOutputDirectory + WriteSHA512PackageHash OR builder.Save

your current code will cause the nupkg to be written out too which i believe is not what you want.

Reply:
writting the nupkg out is exactly what I want. Here is the desired layout which the code currently does:

microsoft.nuget.packagingexample.nupkg
microsoft.nuget.packagingexample.nupkg.sha512
microsoft.nuget.packagingexample.nuspec

Here is the documentation for the feed layout I am targeting.
https://blog.nuget.org/20150922/Accelerate-Package-Source.html

Reply:
Does this resolve your comment?